src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '/src/styles/blocks/variables';
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__total {
+    display: flex;
+  }
+
+  &__number-of-reviews {
+    color: rgba(6, 11, 53, 1);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__total--n div:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
Verify that the images 'star-active.svg' and 'star.svg' are available at the specified paths. Incorrect paths will result in missing images in the UI.